sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -648,7 +648,7 @@ def __init__(self, side_input):
 
 class DataflowPipelineResult(PipelineResult):
   """Represents the state of a pipeline run on the Dataflow service."""
-  def __init__(self, job, runner):
+  def __init__(self, job, options, runner):
```

Comment:
I think we should do this `__init__(self, job, runner, options)` to avoid the potential breaking changes.

Reply:
Yea it would have made my change smaller too. FWIW this class is not intended for public. I may be rusty on how `__all__` works. But this class definitely should only be constructed from within the DataflowRunner.

Reply:
We could update the docstrings to mention this is for internal use and no back-compatible guarantee.